waspc/examples/todoApp/ext/pages/auth/Login.js

```diff
@@ -1,16 +1,24 @@
 import React, { useState } from 'react'
 import { Link } from 'react-router-dom'
+import { useQuery } from '@wasp/queries'
 
 import LoginForm from '@wasp/auth/forms/Login'
+import { getNumTasks } from '@wasp/queries/getNumTasks.js'
```

Comment:
This seems to be the problem -> you need to use default import!
So it should be

```js
import getNumTasks from '@wasp/queries/getNumTasks'
```

That is just how the Wasp generates the code.

I can see thought that this can be confusing, since you defined your function as named export and you could have assumed that it would be named export again. In the future we will probably work around making this more intuitive, so that you will import the function very similarly, or even the same, as is the path to the file where you defined it.

Reply:
@saatvikshah